sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
another _benefit_ of having distinct `hPa` is that it equals `millibar`, which is why `hPa` is commonly preferred over using `kPa` (or plain `Pa`).